script/install/script.rb

```diff
@@ -44,26 +72,58 @@ def copy_theme_files
   cp_start_bootstrap_dir_to_target_assets_dir('vendor/datatables', 'javascripts')
   cp_start_bootstrap_dir_to_target_assets_dir('vendor/jquery', 'javascripts')
   cp_start_bootstrap_dir_to_target_assets_dir('vendor/jquery-easing', 'javascripts')
-  cp_start_bootstrap_dir_to_target_assets_dir('vendor/fontawesome-free/css', 'stylesheets')
   cp_start_bootstrap_dir_to_target_assets_dir('vendor/fontawesome-free/webfonts', 'images')
+
+  cp_fontawesome_css_files
+
+  delete_map_files
```

Comment:
I'm not sure yet how to include the JS map files for precompilation. Since they're not critical, I'm not including them for now.

Reply:
@bigxiang Hmmm... I'm not sure that's the case here. We're importing the minified JS files from the downloaded Start Bootstrap Admin Theme 2 package. When the Rails asset pipeline compiles the minified JS files, I don't think it will be able to map them to the original JS files. Am I missing something?

Reply:
Yeah, so my question is why we are importing the minified JS files to Rails asset pipeline. The asset pipeline is using sprocket which uses one kind of JS compressor at back end. It can generate similar result like what other JS build systems do to generate minified JS files. 

So I think it should be good enough to just place the normal JS files into the asset pipeline. If we need to reference to JS files directly, such as from a CDN, it will be the better case to use minified JS files.

What do you think?

Reply:
You're right. I chose to include the minified files since they made it easier to automate the import of assets and demo files. 

Currently, I think it should be possible to point the demo files to the normal assets, though I haven't tested this yet.

I'll create an issue to remove the minified files from the gem.